Assets/Plugins/Android/mainTemplate.gradle

```diff
@@ -43,9 +43,4 @@ android {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     compile(name: 'shopify_buy_plugin', ext:'aar')
-
-    compile 'com.android.support:appcompat-v7:26.0.0'
-    compile 'com.android.support:customtabs:26.0.0'
-    compile 'com.shopify.mobilebuysdk:buy3-pay-support:1.0.0'
-    compile 'com.squareup.moshi:moshi:1.5.0'
```

Comment:
we don't need these?
`'com.shopify.mobilebuysdk:buy3-pay-support:1.0.0'`
`compile 'com.squareup.moshi:moshi:1.5.0'`

Reply:
Turns out we do. I ran the example app without these and got `ClassNotFound` errors when trying to checkout out.

Now that I think of it, even if we exclude `support-annotations` from everything, if the plugin is added to a project that doesn't have another plugin that has it then that will also be missing which might cause issues within the dependencies.

I think the only way around all of this is trying to figure out how the ill-named https://github.com/googlesamples/unity-jar-resolver works. Looks like Facebook is using this for the same reason since when you pull in that .unitypackage it pulls out the dependency .jars into the plugins folder so the plugins can all use the same one instead of having duplicates.

I'm going to close this issue for now.